python/lsst/pipe/tasks/read_curated_calibs.py

```diff
@@ -25,8 +26,12 @@ def read_one_chip(root, chip_name, chip_id):
         A dictionary of objects constructed from the appropriate factory class.
         The key is the validity start time as a `datetime` object.
     """
-    factory_map = {'qe_curve': Curve, 'defects': Defects}
-    files = glob.glob(os.path.join(root, chip_name, '*.ecsv'))
+    factory_map = {'qe_curves': Curve, 'defects': Defects, 'linearizer': Linearizer}
```

Comment:
You need to turn this back to qe_curve.  It's a rebase error (it used to be qe_curves but we changed it to qe_curve and on rebase you reverted that change).